x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/parser/StatementParserTests.java

```diff
@@ -676,20 +676,30 @@ public void testLikeRLike() {
 
     public void testEnrich() {
         assertEquals(
-            new Enrich(EMPTY, PROCESSING_CMD_INPUT, new Literal(EMPTY, "countries", KEYWORD), new EmptyAttribute(EMPTY), null, List.of()),
+            new Enrich(
+                EMPTY,
+                PROCESSING_CMD_INPUT,
+                null,
+                new Literal(EMPTY, "countries", KEYWORD),
+                new EmptyAttribute(EMPTY),
+                null,
+                List.of()
+            ),
             processingCommand("enrich countries")
         );
 
+        Enrich.Mode mode = randomFrom(Enrich.Mode.values());
         assertEquals(
             new Enrich(
                 EMPTY,
                 PROCESSING_CMD_INPUT,
+                mode,
                 new Literal(EMPTY, "countries", KEYWORD),
                 new UnresolvedAttribute(EMPTY, "country_code"),
                 null,
                 List.of()
             ),
-            processingCommand("enrich countries ON country_code")
+            processingCommand("enrich [mode=" + mode.name() + "] countries ON country_code")
```

Comment:
Not really happy with the naming. Mode is fairly abstract - I would have preferred resolution, resolved, policy-location but it doesn't fit well with `any`, `relative` or `originator`.
Naming is hard. /cc @nik9000  for more suggestions.

Reply:
`from`?